back/src/Services/SocketManager.ts

```diff
@@ -130,7 +130,9 @@ export class SocketManager {
                 );
             });
         } else {
-            emitError(user.socket, "WAM file url is undefined. Cannot edit map without WAM file.");
+            if (!room.roomUrl.match(`#${PLAY_URL}/_/.*#`)) {
+                emitError(user.socket, "WAM file url is undefined. Cannot edit map without WAM file.");
+            }
```

Comment:
Why do we need to do that? The parsing of the URL should really be done only in the admin (or the LocalAdmin class of the pusher). Anything outside looks extremely suspicious to me.